turbo.json

```diff
@@ -0,0 +1,25 @@
+{
+  "ui": "stream",
+  "tasks": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": ["dist/**", "cache/**", "types/**", "release/**"]
```

Comment:
This doesn't cover the outputs for some packages. For example:
- solidity has artifacts and typechain-generated typescript
- helloworld is similar
- starknet has release

Should we add package-specific `turbo.json` for these?

Reply:
i was debating it in my head but overall I think it's the right call, added specific turbo.jsons for solidity/helloworld/starknet here 981b1092a3f7b2ecfb8acd75987cead9237960d0